README.md

```diff
@@ -51,6 +51,38 @@ webdav /some/path/match/* {
 }
 ```
 
+If you want to serve WebDAV and directory listing under same path, e.g. similer behaviour as in Apache and Nginx, you may use [Request Matchers](https://caddyserver.com/docs/caddyfile/matchers) to filter out GET request and pass it to [file_server](https://caddyserver.com/docs/caddyfile/directives/file_server).
+
+Example with authenticated WebDAV and directory listing under the same path:
+
+```
+@notget {
+    not method GET
```

Comment:
Any special reason that `not method GET` is used instead of `method GET` (and then applying it to the `file_server` instead)?

Reply:
Just personal preference, no special reason.

I will leaving both "get" and "not get" macher here for demonstration purpose.